recipes/python-rocksdb/conda_build_config.yaml

```diff
@@ -0,0 +1,14 @@
+c_compiler: gcc
+cpu_optimization_target: nocona
+cxx_compiler: gxx
+fortran_compiler: gfortran
+ignore_build_only_deps: [python]
```

Comment:
This seems ill formed.  It might be interpreted as a selector

Reply:
Removed file